test/repro/reg_test_5363_admin_monitor_caching_sha2.bash

```
@@ -0,0 +1,228 @@
+       NOT running the binary you just built. Re-run with COLD_START=1, or refresh
+       the container with:
+         WORKSPACE=\$(pwd) INFRA_ID=${INFRA_ID} TAP_GROUP=${TAP_GROUP} \\
+           test/infra/control/start-proxysql-isolated.bash"
+fi
+echo "  ${CTR} at ${IP} -- $(adm 'SELECT @@admin-version;')"
+
+restore() {
+	setvar 'admin-admin_credentials' "$BASE_CREDS" 'LOAD ADMIN VARIABLES TO RUNTIME;'
+	setvar 'mysql-default_authentication_plugin' 'mysql_native_password' 'LOAD MYSQL VARIABLES TO RUNTIME;'
+	setvar 'mysql-monitor_password' 'monitor' 'LOAD MYSQL VARIABLES TO RUNTIME;'
+}
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)